_docs_use-cases/use-case-consumption-change-default-repositoryattribute.md

```diff
@@ -1,7 +1,7 @@
 ---
 # Start automatic generation
 permalink: use-case-consumption-change-default-repositoryattribute
-published: false
+published: true
```

Comment:
Please check that the Excel table is up-to-date regarding this use case and run the script to ensure the metadata of the file is set correctly.